src/yupSchema.js

```diff
@@ -213,7 +220,7 @@ const yupSchemasToJsonTypes = {
   object: yupSchemas.fieldset,
   array: yupSchemas.multiple.select,
   boolean: yupSchemas.checkboxBool,
-  null: noop,
+  null: yupSchemas.null,
```

Comment:
todo: Pls create a dev release for us to run the internals tests. Once green, then we can approve/merge this :D

Reply:
On it!